src/models/callercalleeproxy.h

```diff
@@ -46,3 +50,24 @@ class CallerCalleeProxy : public QSortFilterProxyModel
 
 private:
 };
+
+template<typename Model>
+class SourceMapProxy : public CallerCalleeProxy<Model>
```

Comment:
this shouldn't be a template anymore, but instead hardcode Model == SourceMapModel